xendit-android/src/main/java/com/xendit/Xendit.java

```diff
@@ -752,9 +752,13 @@ private void _createAuthenticationWithSessionId(final String tokenId, final Stri
         NetworkHandler handler = new NetworkHandler<Authentication>().setResultListener(new ResultListener<Authentication>() {
             @Override
             public void onSuccess(Authentication responseObject) {
-                if (responseObject.getAuthenticationTransactionId() == null || responseObject.getStatus().equalsIgnoreCase("VERIFIED") || responseObject.getStatus().equalsIgnoreCase("FAILED")) {
-                    // Fallback to 3DS1 flow
+                if (responseObject.getStatus().equalsIgnoreCase("VERIFIED")) {
+                    // Authentication completed
```

Comment:
I think we need to handle if the status id `FAILED `. `4111111111111111` will return `FAILED` at enrollment step for 3D SEMV